static/css/theme.css

```diff
@@ -52,6 +52,6 @@ article [itemprop="description"] {
     font-size: 1em;
   }
   .container {
-    max-width: 50rem;
+    max-width: 75rem;
```

Comment:
I strongly disapprove of extending the content well -> it was explicitly defined 50rem, as a function of font size to make sure that single line would contain no more than 80 characters therefore providing great readability experience.